### ISSUE TITLE
Request functionality: Keyboard shortcut to display all items in the tab dropdown list.

### ISSUE THREAD

**Issue by Tracz-Tech** [holzschu/Carnets#132]
This is a request for functionality. Perhaps it exists already (,in which case I apologise for wasting time), but I can't find it.

When pressing the tab key to show the available methods and attributes the dropbox only displays one item. Clicking on the dropbox expands and reveals all items, however a shortcut to open the dropbox and cycle through the options would be convenient.

I am just learning python and Carnet it my only experience of Jupyter notebooks, but from videos I can see that other systems seem to default to opening up the dropdown list. As an absolute novice, of one week, this appears very useful.

**Comment by holzschu** [comment]
I agree, this would be nice. But this part is managed by the system, and I haven't found a way to do it. It's related to the fact that this is a touchscreen, so you are supposed to touch menus. 
If you have an external keyboard, the up and down arrows cycle through the menu item; otherwise, typing the letters of the method you are looking for will narrow the search.
